### PR TITLE
Inject block into custom-fetchers.

### PR DESCRIPTION
**Pull request by whitefire**
Allows for more versatile item-fetchers.

### CODE REVIEW

File: packages/ezflow_extension/ezextension/ezflow/classes/ezflowoperations.php [hunk 1]
```diff
@@ -73,6 +73,7 @@ public static function updateBlockPoolByBlockID( $block, $publishedBeforeOrAt =
         $fetchClassOptions->iniFile = 'block.ini';
         $fetchClassOptions->iniSection = $block['block_type'];
         $fetchClassOptions->iniVariable = 'FetchClass';
+        $fetchClassOptions->handlerParams = compact('block');
```
Comment (lserwatka) [inline packages/ezflow_extension/ezextension/ezflow/classes/ezflowoperations.php]: I would rather pass an array or some kind of struct, as won't be able to introduce more params without BC in future.

**Comment by lserwatka** [comment]
+1

**Comment by yannickroger** [comment]
Reverted in https://github.com/ezsystems/ezflow/commit/da83b2b7ad780c2636463ff81d9fcc52642e2904

The team did not have time to review this before the merge.
@whitefire can you please reopen a pull request so we can all review it.

**Comment by yannickroger** [comment]
Also can you create a jira issue ( https://jira.ez.no ) corresponding to your patch we need it for the release note and QA process.
You can add the reference to the issue in your commit description ( `git commit --amend` )

File: packages/ezflow_extension/ezextension/ezflow/classes/ezflowfetchparameters.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    /** @var array */
+    public $block;
+
+    public function __construct(array $block)
+    {
+        $this->block = $block;
+    }
+}
+
+?>
```
Comment (andrerom) [inline packages/ezflow_extension/ezextension/ezflow/classes/ezflowfetchparameters.php]: nitpick: add a new line here

**Comment by whitefire** [comment]
New pull-request: https://github.com/ezsystems/ezflow/pull/75

**Pull request merged**